docs/080-breaking-changes.rst

```diff
@@ -61,3 +61,8 @@ New Restrictions
 
 * Declarations with the name ``this``, ``super`` and ``_`` are disallowed, with the exception of
   public functions and events.
+
+Command Line and JSON Interfaces
+================================
```

Comment:
In breaking changes for older versions this section used to be called "Interface Changes" so let's keep it that way:

```suggestion

Interface Changes
=================
```

Reply:
@ssi91 Just noticed that this is marked as resolved but has not actually been changed. Could you add the change?

Reply:
Actually I see that all the changes have reverted to previous versions. Looks like you just removed those commits instead of squashing them.

Reply:
oh, yeah, I forgot to pull the commits which were commited here as suggested ones. I will bring them back